src/utils.rs

```diff
@@ -16,6 +16,17 @@ macro_rules! bigint_str {
     };
 }
 
+#[macro_export]
+macro_rules! vm_prime {
+    () => {
+        BigInt::parse_bytes(
+            b"3618502788666131213697322783095070105623107215331596699973092056135872020481",
+            10,
+        )
+        .unwrap()
+    };
```

Comment:
A `lazy_static!` constant makes more sense for this.

Reply:
Yes,
The only thing I don't like about the `lazy_static!` is that we have to use the `VM_PRIME.clone()` in all the tests
https://github.com/lambdaclass/cleopatra_cairo/pull/332/commits/2600683f6541c87f3aea44b975918b66b09123ac